app/org/sagebionetworks/bridge/models/schedules/Activity.java

```diff
@@ -131,6 +140,10 @@ public Builder withPublishedSurvey(String identifier, String guid) {
             this.survey = new SurveyReference(identifier, guid, null);
             return this;
         }
+        public Builder withPublishedSurvey(String guid) {
+            this.survey = new SurveyReference(null, guid, null);
+            return this;
+        }
```

Comment:
To be honest, I don't like "convenience" overloaded methods with fewer args. If we need to add or remove args in the future, this could scale exponentially. Is there any reason we can't just call withPublishedSurvey(null, guid)? The only exception would be if we were planning identifiers and this is what we want our API to look like.

Also, unit test for this?